### PR TITLE
Added auto-migration feature (2nd attempt)

### PR DESCRIPTION
**Pull request by Wind-Explorer**
Previous PR closed automatically due to me resetting the branch (I think)

Current addition is built on top of latest code from the develop branch

### CODE REVIEW

**Comment by Depal1** [comment]
Related to Pull request #40 and Issue #14.

**Comment by hayatofujii** [comment]
I'm trying to devise a nice strategy for my use case... perhaps we should keep io.PlayCover.* as the new "standard" and make me.PlayCover.* as the symlink?

File: PlayCover/View/PlayCoverApp.swift [hunk 1]
```diff
@@ -31,6 +31,9 @@ class AppDelegate: NSObject, NSApplicationDelegate {
 			defaults: ["NSApplicationCrashOnExceptions" : true]
 		)
         LaunchServicesWrapper.setMyselfAsDefaultApplicationForFileExtension("ipa")
+        if !fm.fileExists(atPath: "/Users/\(NSUserName())/Library/Containers/io.playcover.PlayCover/migrate") {
+                    Migrate.migrateData()
+        }
```
Comment (hayatofujii) [inline PlayCover/View/PlayCoverApp.swift:36]: Mind moving this logic to the `Migrate` class?

**Comment by hayatofujii** [comment]
Did a frankenstein test with this and PR #40 merged. If you get something like

symlink and touch

...nothing will explode!

**Comment by Wind-Explorer** [comment]
this feature had been pointed out to be unessential, hence closing it 💀

**Pull request closed without merging**